### DISCUSSION TITLE
Line chart with time series data of 24 hours

### DISCUSSION THREAD

**Discussion by Murali-001** [Budibase/budibase#9494] [Help]
How to plot line chart with time series data. For plotting continuous data for 24 hours how specify to label column? when I specify time data column as label column, label looking messy. How to make X-axis time table looks interval of time (i.e 0, 4, 8, 12, 16, 20, 24)?

**Comment by melohagan** [comment]
Hey @Murali-001 

Could you share some screenshots please?
